code/modules/security_levels/security_levels.dm

```diff
@@ -60,8 +60,7 @@ GLOBAL_DATUM_INIT(security_announcement_down, /datum/announcement/priority/secur
 
 				var/obj/machinery/door/airlock/highsecurity/red/R = locate(/obj/machinery/door/airlock/highsecurity/red) in GLOB.airlocks
 				if(R && is_station_level(R.z))
-					R.locked = FALSE
-					R.update_icon()
+					R.lock(TRUE)
```

Comment:
Behavior change? Setting locked to true now instead of false. Same with `R.lock(TRUE)` on line 75.